javascripts/discourse/templates/components/top-contributors.hbs

```diff
@@ -10,7 +10,11 @@
         class="top-contributors--user-badge"
       >
         {{avatar item.user imageSize="small"}}
-        {{html-safe item.user.username}}
+        {{#if (and item.user.name siteSettings.enable_names)}}
```

Comment:
It may feel like a regression if someone is expecting usernames here and it suddenly changes...

It may be a stretch, but we could still show usernames if `prioritize username in ux` is enabled. WDYT?

Reply:
Oh yes definitely! Good call. I conflated that setting with `enable_names`